test/e2e_tests/utils/userActions.ts

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
File for shared code for user actions. If it grows too mich we need to split the functions in different files